operations/jsonnet/microservices/component.libsonnet

```diff
@@ -0,0 +1,87 @@
+{
+
+  local k = import 'k.libsonnet',
+  local kausal = import 'ksonnet-util/kausal.libsonnet',
+
+  // local k = import 'ksonnet-util/kausal.libsonnet',
+  local deployment = k.apps.v1.deployment,
+  local container = k.core.v1.container,
+  local volumeMount = k.core.v1.volumeMount,
+  local containerPort = k.core.v1.containerPort,
+  local volume = k.core.v1.volume,
+  local statefulset = k.apps.v1.statefulSet,
+
+  newTempoComponent(target, image='grafana/tempo:latest', port=3200): {
+    local this = self,
+
+    configVolumeName:: 'tempo-conf',
+    overridesVolumeName:: 'overrides',
+
+    containerArgs:: {
+      target: target,
+      'config.file': '/conf/tempo.yaml',
+      'mem-ballast-size-mbs': $._config.ballast_size_mbs,
+    },
+
+    workload: {},
```

Comment:
Since we're not responsible for realizing the workoad, we don't need this key and can instead reference the `target.deployment` directly, or the `target.statefulset` directly.